src/openrct2/PlatformEnvironment.cpp

```diff
@@ -124,35 +124,38 @@ std::unique_ptr<IPlatformEnvironment> OpenRCT2::CreatePlatformEnvironment()
 
     // Set default paths
     std::string basePaths[DIRBASE_COUNT];
-    basePaths[(size_t)DIRBASE::OPENRCT2] = Platform::GetInstallPath();
-    basePaths[(size_t)DIRBASE::USER] = Path::Combine(Platform::GetFolderPath(SPECIAL_FOLDER::USER_DATA), subDirectory);
-    basePaths[(size_t)DIRBASE::CONFIG] = Path::Combine(Platform::GetFolderPath(SPECIAL_FOLDER::USER_CONFIG), subDirectory);
-    basePaths[(size_t)DIRBASE::CACHE] = Path::Combine(Platform::GetFolderPath(SPECIAL_FOLDER::USER_CACHE), subDirectory);
-    basePaths[(size_t)DIRBASE::DOCUMENTATION] = Platform::GetDocsPath();
+    basePaths[static_cast<size_t>(DIRBASE::OPENRCT2)] = Platform::GetInstallPath();
```

Comment:
This does not look like an improvement, or even desirable. Perhaps basePaths should just be a `std::map` instead?

Reply:
Can't say I really agree. The point is to move away from C-style casts to more clearly defined behaviour, not to make it look pretty.

That said, you're welcome to push any improvements to this PR.

Reply:
I'm with @Gymnasiast here, we can change that afterwards, it's a bit out of scope in here